cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillController.java

```diff
@@ -162,11 +164,14 @@ public ResourceReport getResourceReport() {
     String podName = String.format("%s-%d", meta.getName(), instanceId);
 
     try {
-      api.deleteNamespacedPodAsync(podName, kubeNamespace, null, deleteOptions, null, null, null, null,
-                                   createCallbackFutureAdapter(resultFuture, r -> runnable));
+      api.deleteNamespacedPod(podName, kubeNamespace, null, deleteOptions, null, null, null, null);
```

Comment:
Add a TODO to have it revert to async call after CDAP-18000 is fixed.

Reply:
Done